test-pods.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
```

Comment:
You probably didn't intend to commit this, or at least under the root of the project :)